docker/ci

```diff
@@ -12,4 +12,4 @@ docker run $CI_ENV \
            -v $DIR/..:/code \
            -w /code \
            openstax/cookbook.ci:latest \
-           /bin/bash -c "bundle config path vendor/bundle; bundle install; bundle exec rspec"
+           /bin/bash -c "bundle config path vendor/bundle; bundle install; bundle exec rspec spec/kitchen_spec/ || exit; bundle exec rspec spec/recipes_spec/"
```

Comment:
```suggestion
           /bin/bash -c "bundle config path vendor/bundle && bundle install && bundle exec rspec spec/kitchen_spec/ && bundle exec rspec spec/recipes_spec/"
```

It seem that if any of the commands fail then CI should fail. If so, then using `&&` will cause CI to fail before running the next command

Reply:
ooooh thanks phil